### PR TITLE
Basic command manager mappings

### PR DESCRIPTION
**Pull request by modmuss50**
I think I have done this correct. This will allow commands to be added.

### CODE REVIEW

File: mappings/net/minecraft/command/ICommandManager.mapping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLASS none/m net/minecraft/command/ICommandManager
+	METHOD b getCommands ()Ljava/util/Map;
```
Comment (darkevilmac) [inline mappings/net/minecraft/command/ICommandManager.mapping]: I prefer getRegisteredCommands or getManagedCommands. That's more opinion than anything though.

Comment (modmuss50) [inline mappings/net/minecraft/command/ICommandManager.mapping]: Humm, yeah that would make more sense

File: mappings/net/minecraft/command/ICommandManager.mapping [hunk 2]
```diff
@@ -0,0 +1,5 @@
+CLASS none/m net/minecraft/command/ICommandManager
+	METHOD a runCommand (Lnone/n;Ljava/lang/String;)I
```
Comment (modmuss50) [inline mappings/net/minecraft/command/ICommandManager.mapping]: processCommand might be better thinking about it

Comment (asiekierka) [inline mappings/net/minecraft/command/ICommandManager.mapping]: executeCommand?

Comment (shadowfacts) [inline mappings/net/minecraft/command/ICommandManager.mapping]: `execute`, IMO, we already know it's a command

**Comment by modmuss50** [comment]
Ok, this should be ok now

File: mappings/net/minecraft/reference/Commands.mapping [hunk 3]
```diff
@@ -1,4 +1,4 @@
-CLASS none/bc
+CLASS none/bc net/minecraft/reference/Commands
```
Comment (modmuss50) [inline mappings/net/minecraft/reference/Commands.mapping]: Fuck, not this

**Pull request merged**